internal/batches/template/templating.go

```diff
@@ -85,6 +85,31 @@ type BatchChangeAttributes struct {
 	Description string
 }
 
+type TemplatingBranch struct {
+	Name      string
+	TargetOID string
+}
+
+type TemplatingRepository struct {
+	ID            string
+	Name          string
+	DefaultBranch TemplatingBranch
+	FileMatches   map[string]bool
+}
+
+func (r TemplatingRepository) SearchResultPaths() (list fileMatchPathList) {
```

Comment:
Do we need the `Templating` in the name? `template.TemplatingRepository` is a bit redundant.